<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Ekran" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>640</width>
      <height>640</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="wyjdz">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QSlider" name="alpha_poziom">
    <property name="geometry">
     <rect>
      <x>647</x>
      <y>167</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="erozja_b">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>236</y>
      <width>164</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>EROZJA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="domkniecie">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>304</y>
      <width>164</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>DOMKNIĘCIE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>136</y>
      <width>164</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Promień</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="dylatacja_b">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>202</y>
      <width>164</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>DYLATACJA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="otwarcie">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>270</y>
      <width>164</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>OTWARCIE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="black_white">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>black-white</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ekran</class>
   <extends>QWidget</extends>
   <header>ekran.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
